bundles/org.openhab.binding.androiddebugbridge/README.md

```diff
@@ -52,19 +55,33 @@ This is a sample of the mediaStateJSONConfig thing configuration:
 
 `[{"name": "com.amazon.tv.launcher", "mode": "idle"},{"name": "org.jellyfin.androidtv", "mode": "wake_lock", "wakeLockPlayStates": [2,3]},{"name": "com.amazon.firetv.youtube", "mode": "wake_lock", "wakeLockPlayStates": [2]}]`
 
+## Record/Send input events
+As the execution of key events takes a while you can use input events an alternative way to control the device you could use input events. As they are pretty device specific you can use the record-input and recorded-input to store/send those events. 
```

Comment:
... as an alternative.. 
Something strange in the first sentence with the end that looks duplucated. 
Please add comma at right places, that is before the 2 "you can use"
Each sentence one line.